.travis.yml

```diff
@@ -72,6 +80,7 @@ before_install:
   - export PATH="$HOME/miniconda/bin:$PATH"
   - conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
   - conda update conda
+  - conda config --add channels conda-forge --force
```

Comment:
We need this to install `doctr` and other modules.
Ideally we should migrate the `odm2` channel to `conda-forge` to avoid the need for 3 channels.

Reply:
> Ideally we should migrate the odm2 channel to conda-forge to avoid the need for 3 channels.

Thanks for the reminder. Let's discuss it in the near future, say, starting next week (not this week)